examples/standard_platformer/lib/coin.dart

```diff
@@ -5,11 +5,12 @@ import 'package:leap/leap.dart';
 import 'package:tiled/tiled.dart';
 
 class Coin extends PhysicalEntity {
-  final TiledObject tiledObject;
-
-  Coin(this.tiledObject, SpriteAnimation animation)
-      : super(static: true, collisionType: CollisionType.standard) {
-    size = Vector2.all(16);
+  Coin({
+    required this.tiledObject,
+    required SpriteAnimation animation,
+  }) : super(static: true, collisionType: CollisionType.standard) {
+    tiledObject.width = 16;
+    tiledObject.height = 16;
```

Comment:
Does this do anything?

Reply:
Ah I see, this is replacing `size = Vector2.all(16);`, but it's editing the size of the wrong thing, this should be:

```dart
width = 16;
height = 16;
```

The width/height of the `tiledObject` doesn't matter